docs/content/Deployment/Platform-Kubernetes.md

```diff
@@ -0,0 +1,516 @@
+---
+title: Deploy with Kubernetes
+permalink: /deployment/platforms/kubernetes
+category: Deployment
+subCategory: Platforms
+menuOrder: 1
+---
+
+This guide walks you through deploying Cube.js with Kubernetes. This particular
+deployment makes use of a `hostPath` volume to mount schema files into the
+containers.
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| This is an example of a production-ready deployment, but real-world
+| deployments can vary significantly depending on desired performance
+| and scale.
+<!-- prettier-ignore-end -->
+
+## Prerequisites
+
+- [Kubernetes][k8s]
+
+## Configuration
+
+### Create Cube.js API instance and Refresh Worker
+
+We'll start by creating a Redis deployment in a file called
+`redis-deployment.yaml`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: redis
```

Comment:
@adnanrahic Is it safe to change the `io.kompose.service` label to something else?

Reply:
Yep. You can change those to anything you want really.
I tend to keep them as `service`, `name`, or any other sensible label.
Just make sure to edit them **everywhere** so they are the same in every file.

Reply:
I think we should rename all the `io.kompose.service: redis` into `service: redis`.

Reply:
```suggestion
    service: redis
```